test/WebSites/ResponseCacheWebSite/Startup.cs

```diff
@@ -36,6 +36,12 @@ public void ConfigureServices(IServiceCollection services)
                         Duration = 0,
                         Location = ResponseCacheLocation.None
                     });
+
+                options.Filters.Add(new ResponseCacheFilter(new CacheProfile
```

Comment:
- previously were not testing `ResponseCacheFilter` additions
- this filter does _not_ affect any other action because the rest have `[ResponseCache]`